doc/getting_started/getting_started.rst

```diff
@@ -20,22 +20,73 @@ Follow one of the following guides for your host operating system.
    macOS <installation_mac.rst>
    Windows <installation_win.rst>
 
-Clone the Zephyr Repository
-***************************
+.. _get_the_code:
+
+Get the source code
+*******************
+
+Zephyr's multi-purpose :ref:`west` tool lets you easily get the Zephyr project
+source code, instead of manually cloning the Zephyr repos along with west
+itself.
 
-To clone the Zephyr source code repository from GitHub:
+Bootstrap west
+==============
+
+Install the bootstrapper for Zephyr's command-line tool, :ref:`west` in a
+shell or ``cmd.exe`` prompt:
 
 .. code-block:: console
 
-   git clone https://github.com/zephyrproject-rtos/zephyr
+   # Linux
+   pip3 install --user west
```

Comment:
Is it possible to just wget the west bootstrapper?

Reply:
> 
> 
> Is it possible to just wget the west bootstrapper?

check this RFC, and comment whether this is the behavior that you also think `west` should support.
https://github.com/zephyrproject-rtos/west/pull/133

Reply:
> Is it possible to just wget the west bootstrapper?

Yes. We publish wheels and sdists to pypi. You can `wget` the 0.4.0 ones here:

https://files.pythonhosted.org/packages/ba/cf/260bd15d72ca9a9d9bf6006575fb8ef2659dde6334b3f977ca88cc6a9951/west-0.4.0-py3-none-any.whl
https://files.pythonhosted.org/packages/c6/7f/26501ecf53303beac2f8c9729cf9bf25b053d4901674a62b670d91e774a7/west-0.4.0.tar.gz

You can then install them using setuptools or pip or whatever you like.

Reply:
> There was a push to put everything in the "West library".

s/library/package/ then yes. We discussed ad nauseam that package != library.

> It's still independent at least.

And, by design, always will be.

>> You can then install them using setuptools or pip or whatever you like.
> Yes. We publish wheels and sdists to pypi. You can wget the 0.4.0 ones here:
> [...]
> You can then install them using setuptools or pip or whatever you like.

(@aescolar I'm assuming `python setup.py install` on an sdist to a standalone directory that doesn't affect your system path might be acceptable).

The main reason (IMO) why we can't just distribute a Python script has to do with Windows. We want every operating system to be able to run `west <command>` and get the same results. To do that, our setup.py uses `entry_points` to ask for a `west` to be installed in a system-dependent manner.

On Linux, that looks like this:

```
#!/home/mbolivar/.local/share/virtualenvs/west/bin/python3

# -*- coding: utf-8 -*-
import re
import sys

from west._bootstrap.main import main

if __name__ == '__main__':
    sys.argv[0] = re.sub(r'(-script\.pyw?|\.exe)?$', '', sys.argv[0])
    sys.exit(main())
```

On Windows, though, it's a .exe:

```
>>> import shutil
>> shutil.which('west')
'C:\\Python36\\Scripts\\west.EXE'
```

Reply:
> > Is it possible to just wget the west bootstrapper?
> Yes. We publish wheels and sdists to pypi. You can wget the 0.4.0 ones here:

@mbolivar I think that what @aescolar is asking for, it the possibility to just `wget` a single file, and continue from there, just as you do with `repo`.
But that's just how I read his comments, hopefully @aescolar can make his thinking clearer to both of us :)

> The main reason (IMO) why we can't just distribute a Python script has to do with Windows. We want every operating system to be able to run west <command> and get the same results. To do that, our setup.py uses entry_points to ask for a west to be installed in a system-dependent manner.

And that we already achieve with current design.
Question is, should we allow for other ways as well ?
I don't see a blocker or additional cost in also providing the possibility to fetch the bootstrapper as a single file, if there is a wish for that.

Then users can choose their prefered way, while we recommend the `pip` installation.

Reply:
Yes, my comment was about wondering if it would be possible to have such a "local" way of getting the bootstrapper as that of repo. If it is not and it could be added, I think it would be nice for some users. But in this PR context, it was just a question => resolving the conversation.